ar4_hardware_interface/include/ar4_hardware_interface/ar4_servo_gripper_hw_interface.hpp

```diff
@@ -0,0 +1,79 @@
+// BSD 3-Clause License
```

Comment:
If the gripper would still not be integrated, does it make sense to merge it? I'm considering in case someone has an AR4 with a gripper and tries our code.

Reply:
Would it be ok if I add a note in the README stating that the gripper is a work in progress?

Reply:
Yes, that'll work!